chapter09/sxg/ex_9_2.pl

```diff
@@ -0,0 +1,10 @@
+#!perl
+use warnings;
+use strict;
+
+$^I = ".out";
+
+while (<>) {
+    s/Fred/Larry/gi;
+    print;
+}
```

Comment:
I think, I misunderstood the question. Yes it will add a .out at the end of filename.